ui/packages/catalog-ui-search/src/main/webapp/component/query-template-sharing/query-template-sharing.less

```diff
@@ -0,0 +1,111 @@
+@import '../editable-rows/editable-rows';
+
+.icon-label {
+padding-left: 8px;
```

Comment:
Should we use @minimumSpacing here?

Reply:
Setting these to minimum spacing goes against what workspaces has and alters the alignment to be misaligned in this case. I'm open to suggestions or it might make sense to leave this as it is.